hangman.rb

```diff
@@ -0,0 +1,149 @@
+
+class Hangman
+	def initialize
+
+		# Library of words
+		@words = ["orca", "cheese", "cake", "pumpkin", "sun", "apple", "cloud", "paint", "bird", "travel", "lollipop", "ada"]
+
+		@answer_word = choose_word # chooses answer (array of characters)
+		@answer_copy = @answer_word.dup # creates arrage of correct length for @word_display
+		@word_display = @answer_copy.fill("_")
+		@letters_guessed = []
+
+		@default_guesses = 7
+		@wrong_count = 0
+
+		start
+	end
+
+	def start
+		@keep_playing = true
+		puts "Let's play HANGMAN!"
+
+		update_visuals
+		play
+	end
+
+	def play
+		# Checks for win or lose
+		while @keep_playing == true
+			if @wrong_count < 7 && win != true
```

Comment:
Although `win` just seems like a variable here, it is actually a method call. By using the `win` method call here and in line 35 - the code needs to call that method multiple times. I would recommend storing that in a local variable prior to the conditional execution so you don't need to do that.